airflow/upgrade/checker.py

```diff
@@ -94,7 +107,10 @@ def __main__():
     parser = argparse.ArgumentParser()
     register_arguments(parser)
     args = parser.parse_args()
-    args.func(args)
+    if args.list:
+        list_checks()
+    else:
+        run(args)
```

Comment:
```suggestion
    arg.func(args)
```
Right? Otherwise we will have recursion error?

Reply:
D'oh! Thanks, I'll change this

Reply:
Actually, I think the implementation I had was fine, since I removed the `set_defaults` call on the argument parser too. This won't be recursive.

Reply:
Oh, never mind... I missed that it was in `main` function and I thought it was defined in `run`...
<img width="644" alt="Screenshot 2021-01-07 at 07 51 17" src="https://user-images.githubusercontent.com/9528307/103861098-22ee0f80-50bd-11eb-8309-7b0f709f68c0.png">